plugins/woocommerce/tests/api-core-tests/tests/customers/customers-crud.test.js

```diff
@@ -0,0 +1,451 @@
+			request
+		}) => {
+			// call API to retrieve all customers should initially return empty array
+            // unless the role 'all' is passed as a search string, in which case the admin
+            // and subscriber users will be returned
+			const response = await request.get('/wp-json/wc/v3/customers', {
+				params: {
+					role: 'all'
+				}
+			});
+			const responseJSON = await response.json();
+			expect(response.status()).toEqual(200);
+			expect(Array.isArray(responseJSON));
+			expect(responseJSON.length).toBeGreaterThanOrEqual(2);
+		});
+	});
+
+	test.describe('Create a customer', () => {
+		test('can create a customer', async ({
+			request,
+		}) => {
+			// call API to create a customer
+			const response = await request.post('/wp-json/wc/v3/customers', {
+				data: customer,
+			});
+			const responseJSON = await response.json();
+
+			// Save the customer ID. It will be used by the retrieve, update, and delete tests.
+			customerId = responseJSON.id;
+
+			expect(response.status()).toEqual(201);
+			expect(typeof responseJSON.id).toEqual('number');
+			// Verify that the customer role is 'customer'
+			expect(responseJSON.role).toEqual('customer');
+		});
+	});
+
+	test.describe('Retrieve after create', () => {
+		test('can retrieve a customer', async ({
+			request
+		}) => {
+			// call API to retrieve the previously saved customer
+			const response = await request.get(
+				`/wp-json/wc/v3/customers/${customerId}`
+			);
+			const responseJSON = await response.json();
+			expect(response.status()).toEqual(200);
+			expect(responseJSON.id).toEqual(customerId);
+			expect(responseJSON.is_paying_customer).toEqual(false);
+			expect(responseJSON.role).toEqual('customer');
+		});
+
+		test('can retrieve all customers', async ({
+			request
+		}) => {
+			// call API to retrieve all customers
+			const response = await request.get('/wp-json/wc/v3/customers');
+			const responseJSON = await response.json();
+			expect(response.status()).toEqual(200);
+			expect(Array.isArray(responseJSON));
+			expect(responseJSON.length).toBeGreaterThan(0);
+		});
+	});
+
+	test.describe('Update a customer', () => {
+
+		test(`can update the admin user/customer`, async ({
+			request,
+		}) => {
+			// update customer names (regular, billing and shipping) to admin
+			// (these were initialised blank)
+			const response = await request.put(
+				`/wp-json/wc/v3/customers/1`, {
+					data: {
+						first_name: 'admin',
+						billing: {
+							first_name: 'admin'
+						},
+						shipping: {
+							first_name: 'admin'
+						}
+					}
+				}
+			);
+			const responseJSON = await response.json();
+			expect(response.status()).toEqual(200);
+			expect(responseJSON.first_name).toEqual('admin');
+			expect(responseJSON.billing.first_name).toEqual('admin');
+			expect(responseJSON.shipping.first_name).toEqual('admin');
+		});
+
+		test('retrieve after update admin', async ({
+			request
+		}) => {
+			// call API to retrieve all customers
```

Comment:
Just need to update this comment since this is only being called to return the admin customer and not all customers

```suggestion
			// call API to retrieve the admin customer we updated above
```